### DISCUSSION TITLE
How can I automate the selection of best model and create it using create_model()

### DISCUSSION THREAD

**Discussion by Pem14604** [pycaret/pycaret#1636] [Q&A] [answered]
Hi team,
 could you please suggest me how can I get the name of best model form compare_models output and send it to create_model automatically rather then passing manually.

Example
As we can see below "rf" is the best model and I have created it using create_model() by passing the parameter "rf", can I automate this.

or can I get list or dict something from compare_models output as per their accuracy with their name like "rf","dt" so that i can pass them in create_model.

![image](https://user-images.githubusercontent.com/43243814/135436588-19e92cb5-88e6-4dcc-b772-f6cdf94205fd.png)

**Comment by Yard1** [comment, marked as answer]
You can use the `best_model` returned by `compare_models` directly. You don't need to create it again. There is also a parameter in `compare_models` that controls how many of those best models are returned.

**Reply by Pem14604** [reply]
Thanks for the quick reply.
